ocaml/idl/datamodel_host.ml

```diff
@@ -1678,7 +1678,7 @@ let host_numa_affinity_policy =
            using soft-pinning, but the policy doesn't guarantee a balanced \
            placement, falling back to the 'any' policy."
         )
-      ; ( "default"
+      ; ( "default-policy"
```

Comment:
Better use _ instead of -. That's the convention in the API. I see I've missed `best-effort`...

Reply:
So let's also change `best-effort` to `best_effort`...

Reply:
We have 'balance-slb' that has a '-'. I'm fine either way

Reply:
Then that one is bad also :) All enums in `host` and `VM` at least have _

Reply:
A bit too much of a replace-all now I think...